src/common/components/Header/style.less

```diff
@@ -9,3 +9,48 @@
     line-height: @layout-header-height;
   }
 }
+
+.responsive-menu {
```

Comment:
can we get rid of these styles? we should have as least of styles as possible in starter, just reusing basic default ant components

Reply:
Yeah, this was just to keep the implementation the same.
The default will just pop-up the menu in a dropdown, not full screen. Should be enough.